### PR TITLE
Add reproducer tests into guided decision table backend

### PR DESCRIPTION
**Pull request by jomarko**
This PR adds reproducer tests for:
- RHBA-370
- RHDM-329

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/resources/org/drools/workbench/screens/guided/dtable/backend/server/rhba370/src/main/resources/META-INF/kie-deployment-descriptor.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/resources/org/drools/workbench/screens/guided/dtable/backend/server/rhba370/src/main/resources/META-INF/kie-deployment-descriptor.xml]: Do we need this file?

Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/resources/org/drools/workbench/screens/guided/dtable/backend/server/rhba370/src/main/resources/META-INF/kie-deployment-descriptor.xml]: Well spotted, removed now.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@manstis jenkins says, green.

**Pull request merged**